### ISSUE TITLE
compare fails with Key Error

### ISSUE THREAD

**Issue by bhishanpdl** [fbdesignpro/sweetviz#42]
This seems to be a possible bug:
```
Date:  Jul 22, 2020
platform: Macos
environment: conda custom environment
sweetviz version:  sweetviz==1.0b3

np.__version__ # 1.18.4
pd.__version__ # 1.0.3
```

# MWE
```python
import numpy as np
import pandas as pd
import seaborn as sns

import sweetviz

df = sns.load_dataset('titanic')
my_report = sweetviz.compare(
    [df.query("sex == 'male'"), "Male"],
    [df.query("sex == 'female'"), "Female"],
    "survived"
    )

```

# Error
```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/pandas/core/series.py in _set_value(self, label, value, takeable)
   1138             else:
-> 1139                 self.index._engine.set_value(self._values, label, value)
   1140         except (KeyError, TypeError):

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.set_value()

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.set_value()

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.get_loc()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable.get_item()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable.get_item()

KeyError: True

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
<ipython-input-62-d04e4b8c2811> in <module>
      2     [df.query("sex == 'male'"), "Male"],
      3     [df.query("sex == 'female'"), "Female"],
----> 4     "survived"
      5     )
      6 

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/sweetviz/sv_public.py in compare(source, compare, target_feat, feat_cfg, pairwise_analysis)
     21             pairwise_analysis: str = 'auto'):
     22     report = sweetviz.DataframeReport(source, target_feat, compare,
---> 23                                       pairwise_analysis, feat_cfg)
     24     return report
     25 

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/sweetviz/dataframe_report.py in __init__(self, source, target_feature_name, compare, pairwise_analysis, fc)
    215             # start = time.perf_counter()
    216             self.progress_bar.set_description(':' + f.source.name + '')
--> 217             self._features[f.source.name] = sa.analyze_feature_to_dictionary(f)
    218             self.progress_bar.update(1)
    219             # print(f"DONE FEATURE------> {f.source.name}"

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/sweetviz/series_analyzer.py in analyze_feature_to_dictionary(to_process)
     97             # Explicitly show missing categories on each set
     98             if compare_type == FeatureType.TYPE_CAT or compare_type == FeatureType.TYPE_BOOL:
---> 99                 fill_out_missing_counts_in_other_series(to_process.compare_counts, to_process.source_counts)
    100                 fill_out_missing_counts_in_other_series(to_process.source_counts, to_process.compare_counts)
    101         returned_feature_dict["compare"] = dict()

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/sweetviz/series_analyzer.py in fill_out_missing_counts_in_other_series(my_counts, other_counts)
     43                 if my_counts[to_fill].index.dtype.name == 'category':
     44                     my_counts[to_fill] = my_counts[to_fill].reindex(my_counts[to_fill].index.add_categories(key))
---> 45                 my_counts[to_fill].at[key] = 0
     46 
     47 def add_series_base_stats_to_dict(series: pd.Series, counts: dict, updated_dict: dict) -> dict:

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/pandas/core/indexing.py in __setitem__(self, key, value)
   2192         key = list(self._convert_key(key, is_setter=True))
   2193         key.append(value)
-> 2194         self.obj._set_value(*key, takeable=self._takeable)
   2195 
   2196 

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/pandas/core/series.py in _set_value(self, label, value, takeable)
   1140         except (KeyError, TypeError):
   1141             # set using a non-recursive method
-> 1142             self.loc[label] = value
   1143 
   1144         return self

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/pandas/core/indexing.py in __setitem__(self, key, value)
    669             key = com.apply_if_callable(key, self.obj)
    670         indexer = self._get_setitem_indexer(key)
--> 671         self._setitem_with_indexer(indexer, value)
    672 
    673     def _validate_key(self, key, axis: int):

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/pandas/core/indexing.py in _setitem_with_indexer(self, indexer, value)
    870         else:
    871 
--> 872             indexer, missing = convert_missing_indexer(indexer)
    873 
    874             if missing:

~/opt/miniconda3/envs/dataSc/lib/python3.7/site-packages/pandas/core/indexing.py in convert_missing_indexer(indexer)
   2342 
   2343         if isinstance(indexer, bool):
-> 2344             raise KeyError("cannot use a single bool to index into setitem")
   2345         return indexer, True
   2346 

KeyError: 'cannot use a single bool to index into setitem'

```

**Comment by fbdesignpro** [comment]
@bhishanpdl should be fixed with the latest 1.0beta4, let us know! :)

**Comment by bhishanpdl** [comment]
The problem is solved.
Date: Aug 2, 2020
```
!/Users/poudel/opt/miniconda3/envs/viz/bin/pip show sweetviz

Name: sweetviz
Version: 1.0b5
Summary: A pandas-based library to visualize and compare datasets.
Home-page: https://github.com/fbdesignpro/sweetviz
Author: Francois Bertrand
Author-email: fb@fbdesignpro.com
License: MIT
Location: /Users/poudel/opt/miniconda3/envs/viz/lib/python3.7/site-packages
Requires: numpy, scipy, tqdm, importlib-resources, jinja2, matplotlib, pandas
Required-by: 

```
